product_specifications/12_xfomv_e/detail.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Deactivation of repository
+
+        Background:
+            Given that a Consortium Lead or Member cannot find a suitable host
+
+        Scenario: Fabrica de-activation
+            Given that a staff_admin user is in the Consortium Lead or Member
+             When the user select in-active
+             Then the member and all its repositories are to be labeled in-active in both Fabrica and the REST API
+              And all the DOIs of the member should be set to 'registered'
+
+Feature: Thombstone page for DOIs with broken links of inactive repositories
+
+        Background:
+            Given that a Repository has been set inactive
+              And that the DOI was set to 'registered' state by inactive workflow
+        
+        Scenario: set tombstone page 
```

Comment:
@sarala I am not sure if this is the scenario to be described here or the feature to be described. 

This sounds like a different feature. something along the lines of : "inactive repository status page" kind of feature.